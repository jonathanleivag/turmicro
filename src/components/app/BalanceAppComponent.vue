<script setup lang="ts">
import CardUiComponent from '../ui/CardUiComponent.vue'
</script>

<template>
  <CardUiComponent title="Balance" />
</template>
