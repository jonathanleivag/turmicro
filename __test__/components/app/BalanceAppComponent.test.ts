import { mount } from '@vue/test-utils'

import BalanceAppComponent from '../../../src/components/app/BalanceAppComponent.vue'

describe('BalanceAppComponent', () => {
  const wrapper = mount(BalanceAppComponent)

  test('Snapshot de BalanceAppComponent', () => {
    expect(BalanceAppComponent).toBeTruthy()
    expect(wrapper.html()).toMatchSnapshot()
  })

  test('El titulo tiene que ser Balance', () => {
    expect(wrapper.find('h3').text()).toBe('Balance')
  })
})
