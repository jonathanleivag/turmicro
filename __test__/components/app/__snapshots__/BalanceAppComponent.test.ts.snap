// Vitest Snapshot v1

exports[`BalanceAppComponent > Snapshot de BalanceAppComponent 1`] = `
"<div class=\\"card\\">
  <div class=\\"card__header\\">
    <h3 class=\\"card__title\\">Balance</h3>
  </div>
  <div class=\\"card__body\\"></div>
</div>"
`;
